style.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+}
+
+.wrapper {
+  background-image: url("./images/headerBackgd.png");
+  background-size: cover;
+  width: 100%;
+  max-width: 500px;
+  padding-bottom: 90px;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  background-color: rgb(245, 250, 245);
+  height: auto;
+  display: flex;
+  padding-top: 15px;
+  margin-bottom: 100px;
+  padding-bottom: 10px;
+}
```

Comment:
- [ ] It would be nicer to change the background of your header to white color so your design matches the Figma design.
Your design:
<img width="211" alt="image" src="https://user-images.githubusercontent.com/77190203/154270453-bd43a28f-80f9-456e-b70b-96c5b5422803.png">